src/test.rs

```diff
@@ -501,3 +501,69 @@ fn var_drop_used_simple() {
         tables.points.untern(first_defined)
     );
 }
+
+/// This test ensures one of the two placeholder origins will flow into the
+/// other, but without declaring this subset as a `known_subset`, which is
+/// an illegal subset relation error.
+#[test]
+fn illegal_subset_error() {
+    let program = r"
+        universal_regions { 'a, 'b }
+        placeholder_loans { 'a: L1, 'b: L2 }
```

Comment:
This feels tedious. I guess our new formulation would make this just 

```
placeholders { 'a: L1, 'b: L2 }
```

but I wonder if we can just write 

```
placeholders { 'a, 'b }
```

and use the `'a` names for the loans too?

Reply:
yeah, that was the initial plan, this WIP state is to avoid most changes to `universal_regions` everywhere to focus on subset errors at first, and then clean this up

Reply:
I've done this as we talked about: in the program syntax `placeholders { 'a, 'b }` replaces both `universal_regions { 'a, 'b }` and `placeholder_loans {'a: L1, 'b: L2}`, and is used to fill both the `universal_region` and `placeholder` input relations (until the redundant `universal_region` is removed everywhere, in rustc and the `.facts` files)